app/models/motion.rb

```diff
@@ -19,6 +19,7 @@ class Motion < ActiveRecord::Base
   has_many :did_not_voters, through: :did_not_votes, source: :user
   has_many :events,        -> { includes(:eventable) }, as: :eventable, dependent: :destroy
   has_many :motion_readers, dependent: :destroy
+  has_many :specifics, as: :specifiable
```

Comment:
No, specifics were put in to make reactions nice and easy.

I reckon there's a bunch of use cases that this covers though, and don't really want to encourage people to put tables into their instance because they'll have no easy way to remove them.